modules/ROOT/pages/change-custom-policy-mule3.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager, you can change the version of your custom policies for your Mule 3 applications without having to remove your current policies leaving your API instance insecure.
+In API Manager, you can change the version of your custom policies for your Mule 3 applications without having to remove your current policies and leaving your API instance insecure.
```

Comment:
and leaving -> and leave

Reply:
True...not parallel. Thanks for catching that.